sdk/eventhub/Microsoft.Azure.EventHubs/tests/Processor/ProcessorTestBase.cs

```diff
@@ -18,398 +18,446 @@ namespace Microsoft.Azure.EventHubs.Tests.Processor
 
     public class ProcessorTestBase
     {
-        protected string LeaseContainerName;
-        protected string[] PartitionIds;
-
-        public ProcessorTestBase()
+        //Unused after Track two test infrastructure ported
+        //protected string LeaseContainerName;
+        //protected string[] PartitionIds;
+
+        //Unused after Track two test infrastructure ported
+        //public ProcessorTestBase()
+        //{
+        //    // Use entity name as lease container name.
+        //    // Convert to lowercase in case there is capital letter in the entity path.
+        //    // Uppercase is invalid for Azure Storage container names.
+        //    this.LeaseContainerName = new EventHubsConnectionStringBuilder(TestUtility.EventHubsConnectionString).EntityPath.ToLower();
+
+        //    // Discover partition ids.
+        //    TestUtility.Log("Discovering partitions on eventhub");
+        //    var ehClient = EventHubClient.CreateFromConnectionString(TestUtility.EventHubsConnectionString);
+        //    var eventHubInfo = ehClient.GetRuntimeInformationAsync().WaitAndUnwrapException();
+        //    this.PartitionIds = eventHubInfo.PartitionIds;
+        //    TestUtility.Log($"EventHub has {PartitionIds.Length} partitions");
+        //}
+
+        //update method to use EventHub managed by EventHubScope
+        public String[] ProcessorPartitionIds(string connectionString)
         {
-            // Use entity name as lease container name.
-            // Convert to lowercase in case there is capital letter in the entity path.
-            // Uppercase is invalid for Azure Storage container names.
-            this.LeaseContainerName = new EventHubsConnectionStringBuilder(TestUtility.EventHubsConnectionString).EntityPath.ToLower();
-
-            // Discover partition ids.
-            TestUtility.Log("Discovering partitions on eventhub");
-            var ehClient = EventHubClient.CreateFromConnectionString(TestUtility.EventHubsConnectionString);
+            var ehClient = EventHubClient.CreateFromConnectionString(connectionString);
             var eventHubInfo = ehClient.GetRuntimeInformationAsync().WaitAndUnwrapException();
-            this.PartitionIds = eventHubInfo.PartitionIds;
-            TestUtility.Log($"EventHub has {PartitionIds.Length} partitions");
+           String[] PartitionIds = eventHubInfo.PartitionIds;
```

Comment:
Not sure why this intermediate variable is needed.   Couldn't we simply return `eventHubInfo.PartitionIds`?